automations/js/src/count_user_reviewable_prs.js

```diff
@@ -49,12 +50,15 @@ query ($repoOwner: String!, $repo: String!, $cursor: String) {
     '🟥 priority: critical',
   ]
   const [owner, repo] = GITHUB_REPOSITORY.split('/')
+  const isValidPR = (pr) =>
```

Comment:
Just a nit to suggest "relevant" rather than "valid". Valid makes it sound like we're asking whether the PR is a real PR, but relevant captures what we actually want to know: whether it's a PR we care to evaluate in this context.